docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -755,24 +755,207 @@ To read and write a value of the Bluetooth descriptor:
 ### Accessing connection's ATT MTU
 <!-- END #11 -->
 
-<!-- TODO #12
-      - write about registerService()
- -->
```

Comment:
We leave this part in other PR for convenience of merging them all at the end.

Reply:
Agree wiith Arek. We will remove commented TODO sections in the last commit, after all other PRs will be merged.
Please do not remove any of them in your changes,

Reply:
Done